README.md

```diff
@@ -50,13 +50,24 @@ pipx run --spec phylum phylum-ci <options>
 These installation methods require Python 3.9+ to run.
 For a self contained environment, consider using the Docker image as described below.
 
-A Windows binary, `phylum-ci.exe`, is offered as a [release artifact][latest_rels] and does not require Python to run.
-Simply [download the latest version][latest_artifact] and run it to access the same
-[`phylum-ci` script entry point features][anchor_script].
+Windows binaries are offered as [release artifacts][latest_rels] for a "standalone" solution that does not require
+Python or Docker to run. There are two options for this installation method:
+
+* `phylum-ci.zip`
+  * [Download the latest version][latest_zip] and extract the archive
+  * Add the extracted directory to `PATH` or reference the contained `phylum-ci.exe` binary directly
+* `phylum-ci.exe`
+  * [Download the latest version][latest_exe] and place this binary on `PATH` or reference it directly
```

Comment:
Should we change this to "Download the latest archive/executable"? Seems a bit odd to have download the latest version twice and only have an explanation for the difference later on without ever explicitly pointing that difference out.

If you change it I'd also change it to `... and extract it` probably.

Reply:
We talked offline and made changes together.